docs/manual/forms/examples/auth_static_variables.rst

```diff
@@ -0,0 +1,70 @@
+.. _examples_auth_static_variables:
+
+===========================
+Formulier met authenticatie
+===========================
+
+In dit voorbeeld maken we een fictief formulier bestaande uit 1 stap, waarbij
+logica en static variabelen worden gebruikt om te zien of de gebruiker als persoon of als bedrijf ingelogd is.
```

Comment:
```suggestion
logica en vaste variabelen worden gebruikt om te zien of de gebruiker als persoon of als bedrijf ingelogd is.
```
I think we should use "static variable" -> "vaste variabele" translation to keep it understandable.